docs/user/azure/encryption.md

```diff
@@ -0,0 +1,115 @@
+# Azure Disk Encryption with Customer Managed Keys
+
+The installer can use an existing disk encryption set. The disk encryption set should be created in a separate resource group than the installer to avoid losing access to your keys. 
```

Comment:
"to avoid losing access to your keys after the cluster is destroyed."

Reply:
Fixed.